Plugins/AppVeorBuldReport/Settings.cs

```diff
@@ -0,0 +1,50 @@
+﻿using System.Windows.Forms;
+using GitUIPluginInterfaces;
+
+namespace GitExtensions.Plugins.BuildServer
+{
+    public sealed class Setting : ISetting, ISettingControlBinding
+    {
+        public Setting(string name, BuildServerIntegrationSettingsPage control)
+        {
+            Name = name;
+            Control = control;
+        }
+
+        public string Name { get; }
+
+        public string Caption { get; }
+
+        public BuildServerIntegrationSettingsPage Control { get; }
+
+        public ISettingControlBinding CreateControlBinding()
+        {
+            return this;
+        }
```

Comment:
[nit]
```suggestion
        public ISettingControlBinding CreateControlBinding() => this;
```
ditto below